mne/io/eyetrack/eyetrack.py

```diff
@@ -0,0 +1,243 @@
+
+    Parameters
+    ----------
+    fname : str
+        Path to the data file (.XXX).
+    %(preload)s
+    %(verbose)s
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+    """
+
+    @verbose
+    def __init__(self, fname, preload=False, verbose=None,
+                 annotate_missing=True, interpolate_missing=True,
+                 read_eye_events=True):
+        from ...preprocessing import annotate_nan
+        from ...preprocessing.interpolate import interpolate_nan
+
+        logger.info('Loading {}'.format(fname))
+
+        # load data
+        ftype = fname.split('.')[-1]
+        if ftype == 'asc':
+            sfreq = 500
+            eye = 'BINO'
+            pos = True
+            pupil = True
+
+            info, data, first_sample, annot, meas_date = \
+                self._parse_eyelink_asc(
+                    fname,
+                    sfreq=sfreq,
+                    eye=eye,
+                    pos=pos,
+                    pupil=pupil,
+                    read_eye_events=read_eye_events)
+        elif ftype == 'edf':
+            raise NotImplementedError('Eyelink .edf files not supported, yet')
+        else:
+            raise ValueError(
+                'This reader can only read eyelink .asc (or .edf) files. '
+                'Got .{} instead'.format(ftype))
+
+        # create mne object
+        super(RawEyelink, self).__init__(  # or just super().__init__( ?
+            info, preload=data, filenames=[fname], verbose=verbose)
+        # set meas_date
+        self.set_meas_date(meas_date)
+        # set annotiations
+        self.set_annotations(annot)
+        # annotate missing data
+        if annotate_missing:
+            annot_bad = annotate_nan(self)
+            self.set_annotations(self.annotations.__add__(annot_bad))
+        # interpolate missing data
+        if interpolate_missing:
+            self = interpolate_nan(self)
+
+    def _parse_eyelink_asc(self, fname, sfreq, eye='BINO', pos=True,
+                           pupil=True, read_eye_events=True):
+        from .ParseEyeLinkAscFiles_ import ParseEyeLinkAsc_
+        import datetime as dt
+
+        # read the header (to extract relevant information)
+        with open(fname, 'r') as f:
+            d_header = []
+            for l in f.readlines()[:100]:  # restrict to first 100 lines
+                d_header.append(l) if ('**' in l) else None
+
+        for l in d_header:
+            if 'DATE:' in l:
+                datetime_str = l.strip('\n').split('DATE: ')[-1]
+            # we can get more, e.g. camera settings
+            if 'CAMERA:' in l:
+                cam = l.strip('\n').split('CAMERA: ')[-1]
+
+        meas_date = None
+        if 'datetime_str' in locals():
+            meas_date = dt.datetime.strptime(datetime_str,
+                                             '%a %b %d %H:%M:%S %Y')
+            meas_date = meas_date.replace(tzinfo=dt.timezone.utc)
```

Comment:
@dominikwelke I'm not sure that this is correct. The date/time string in the .ASC file header appears to be naive (no timezone info). Using `replace` `tzinfo` just forces the timezone string to be 'UTC'. This would only be correct if that naive date/time string is already expressed in UTC, and not a local timezone. We should check with Eyelink (I have been in touch with them and can send an email). 

My other concern is that this date/time string generally won't accurately report the date/time of the session. This date/time string is taken from the Host PC (that runs on the homebrewed QNX OS), which is never connected to the internet. At least in our lab here, the date/time reported in the .asc files is very innacurate. Is that the case for the .asc files in your lab?

Reply:
I agree that we don't want to ask users to specify a timezone via an argument parameter - too many opportunities for mistakes. But if we want to set the `meas_date`, we'll have to specify a time zone or `mne` will return an error. IMO _not_ setting a  `meas_date` is preferable to setting an incorrect one. Anyway, I think the `meas_date` here is primarily metadata, I don't _think_ we will need it for co-registration with EEG/MEG.. We will find out when we start working on that! 

If you send me your email on discord, I can put you in CC in my emails with SR. I'll ask them if there is a good way to discern the timezone in the .asc files.